src/main/learnocaml_main.ml

```diff
@@ -120,12 +125,14 @@ module Args = struct
         Grader_cli.dump_dot := dump_dot;
         Learnocaml_process_exercise_repository.dump_outputs := dump_outputs;
         Learnocaml_process_exercise_repository.dump_reports := dump_reports;
-        { exercises; output_json }
+        ()
```

Comment:
Why did you not remove this statement?

Reply:
I guess it's a style thing, but in this particular codebase it felt natural to insist on the fact that running this term returns the `()` value (for example it explains why we match on this value in the `apply` people below). Would you like me to remove it?